Descriptors/PerformanceLevelDescriptor.xml

```diff
@@ -19,63 +19,69 @@
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Fail</CodeValue>
-		<ShortDescription>Fail</ShortDescription>
-		<Description>Fail</Description>
+		<CodeValue>Well Below Basic</CodeValue>
+		<ShortDescription>Well Below Basic</ShortDescription>
+		<Description>Well Below Basic</Description>
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Pass</CodeValue>
-		<ShortDescription>Pass</ShortDescription>
-		<Description>Pass</Description>
+		<CodeValue>Met Standard</CodeValue>
+		<ShortDescription>Met Standard</ShortDescription>
+		<Description>Met Standard</Description>
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Proficient</CodeValue>
-		<ShortDescription>Proficient</ShortDescription>
-		<Description>Proficient</Description>
+		<CodeValue>Did Not Meet Standard</CodeValue>
+		<ShortDescription>Did Not Meet Standard</ShortDescription>
+		<Description>Did Not Meet Standard</Description>
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Well Below Basic</CodeValue>
-		<ShortDescription>Well Below Basic</ShortDescription>
-		<Description>Well Below Basic</Description>
+		<CodeValue>Pass</CodeValue>
+		<ShortDescription>DEPRECATE: Pass</ShortDescription>
+		<Description>DEPRECATE: Pass</Description>
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Met Standard</CodeValue>
-		<ShortDescription>Met Standard</ShortDescription>
-		<Description>Met Standard</Description>
+		<CodeValue>Fail</CodeValue>
+		<ShortDescription>DEPRECATE: Fail</ShortDescription>
+		<Description>DEPRECATE: Fail</Description>
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Above Benchmark</CodeValue>
-		<ShortDescription>Above Benchmark</ShortDescription>
-		<Description>Above Benchmark</Description>
+		<CodeValue>Commended Performance</CodeValue>
+		<ShortDescription>DEPRECATE: Commended Performance</ShortDescription>
+		<Description>DEPRECATE: Commended Performance</Description>
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Commended Performance</CodeValue>
-		<ShortDescription>Commended Performance</ShortDescription>
-		<Description>Commended Performance</Description>
+		<CodeValue>Above Benchmark</CodeValue>
+		<ShortDescription>DEPRECATE: Above Benchmark</ShortDescription>
```

Comment:
Shouldn't all the DEPRECATE markers here be DEPRECATED?

Reply:
Fixed.